src/main/java/com/mraof/minestuck/util/BlockHitResultUtil.java

```diff
@@ -0,0 +1,43 @@
+package com.mraof.minestuck.util;
+
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.ClipContext;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.phys.BlockHitResult;
+import net.minecraft.world.phys.Vec3;
+import net.minecraftforge.common.ForgeMod;
+
+public class BlockHitResultUtil
+{
+	/**
+	 * Based on the Item class function of the same name, without the FluidMode context
+	 */
+	public static BlockHitResult getPlayerPOVHitResult(Level level, Player playerEntity)
```

Comment:
Why is this needed over the version in `Item`?

Reply:
Its not entirely necessary, all it does is skip the fluid check. It seemed useful to have it, along with the other two functions, be all kept in one place

Reply:
With the fluid check, is it the `ClipContext.Fluid` argument you're referring to? Because it is not hardcoded in the item version of the function.

Reply:
Ah wait I forgot a reason why I did this upon going to delete. The original function has protected access